packages/twenty-front/src/modules/object-record/record-field/states/lastShowPageRecordId.ts

```diff
@@ -0,0 +1,6 @@
+import { createState } from 'twenty-ui';
+
+export const lastShowPageRecordIdState = createState<string | null>({
```

Comment:
this looks to be leaving on its own, I don't get why it's in "record-field" module

Reply:
i would expect this to be in a view-navigation module or something like this